VirtualEconomyFramework/VENFTApp-Blazor/Components/Account/TransactionsList.razor

```diff
@@ -5,7 +5,7 @@
 <div class="row w-100 m-0 d-flex justify-content-center align-items-center">
     @if (TransactionsDetails == null)
     {
-        <p><em><Spin Tip="Loading..." /></em></p>

```

Comment:
Sometime we use italic, sometimes not. I hardcoded it to always use italic.
We can make parameter for that if needed

Reply:
This is OK, lets use italic.

Another thing is, that it display the spinner on my pc like this:

![image](https://user-images.githubusercontent.com/78320021/154149504-ff4a2383-17b0-4ee0-9c55-aac3ec5d0d47.png)

Same in other situations: 

![image](https://user-images.githubusercontent.com/78320021/154149419-4d6877a4-c2b2-4d49-ae22-27bfa6283585.png)


I use Edge (Version 98.0.1108.43) and MSVisualStudio 2019. Do I miss something?

Reply:
Good point, thanks.
I found there is some problem with cache in Edge. We should definitely do some research if blazor have some easy solution for this problem.

Now you can:
1) Open [http://localhost:8000/css/app.css](http://localhost:8000/css/app.css) in browser. You will see no `.spin-overlay` and `.spin-tip` classes)
2) Refresh (Ctrl + R)
3) Should be fixed

Reply:
If we don't find simple solution, we can make some preprocessor.

1) Create template file, for example `template.index.html` with linked css files like this:
```
<link href="css/app.css?v=${version}" rel="stylesheet" />
```
2) Create preprocessor script, that loads content from `template.index.html`, change `${version}` to some unique key (app verison or commit hash) and save result as `index.html`.
3) Execute preprocessor script before build

Reply:
Ok. It was the cache.
The same thing is when you update some external js file. 
I will merge it now, because the issue was for spin component and it is working now. Please feel free to create new issue for the version of the css or js files to force the cache.